README.md

```diff
@@ -99,17 +117,37 @@ The *tsc-agenda* tag is used to elevate items to the next TSC meeting. TSC meeti
 
 Every Working Group must specify a tag that can be used to indicate items for review by the Working Group and a tag that can be used to identify Working Group meeting notes and announcements.
 
-## Release Process
+## Release Process and Branching Model
+
+The Project source repository will be organized into a single `master` branch, multiple maintenance branches, and multiple development branches. All modifications intended for the current major release line must be made to the `master` branch and must follow the guidelines detailed in this document.
 
-### Release Branches
+At each new *semver-minor*, a Release tag is cut (e.g. `v2.0`, `v2.1`, etc).
```

Comment:
Does this mean that patch level releases wouldn't be tagged? This seems contrary to what both projects currently do. See https://github.com/iojs/io.js/releases and https://github.com/joyent/node/releases